app/scheduled/app-client-expire/index.ts

```diff
@@ -46,7 +45,7 @@ export async function handler() {
         'attribute_not_exists(expired) and (lastUsed < :warningCutoff or (attribute_not_exists(lastUsed) and created < :warningCutoff))',
       TableName,
       ExpressionAttributeValues: {
-        ':warningCutoff': warningCutoff,
+        ':warningCutoff': { N: warningCutoff.toString() },
```

Comment:
Are you sure that this is the correct form, that the library does not marshal JavaScript types to DynamoDB types automatically?

Reply:
I'm pretty sure that this should be:

```suggestion
        ':warningCutoff': warningCutoff,
```

Reply:
I think you're right. That is what I had at first, but there was an error, but I no longer get that error when I try this. It may have been just vscode bugging out